.github/workflows/e2e-android.yml

```diff
@@ -17,6 +22,25 @@ jobs:
     # If it takes longer it usually fails, so no need to pay for more
     timeout-minutes: 25
     steps:
+      - uses: google-github-actions/auth@v2
+        if: ${{ env.SECRETS_AVAILABLE }}
+        with:
+          project_id: celo-mobile-mainnet
+          credentials_json: ${{ secrets.MAINNET_SERVICE_ACCOUNT_KEY }}
+      - name: Google Secrets
+        if: ${{ env.SECRETS_AVAILABLE }}
+        id: google-secrets
+        uses: google-github-actions/get-secretmanager-secrets@v2.1.4
+        with:
+          secrets: |-
+            WALLET_CONNECT_PROJECT_ID_E2E:projects/1027349420744/secrets/WALLET_CONNECT_PROJECT_ID_E2E
+            DEFAULT_RECIPIENT_MNEMONIC:projects/1027349420744/secrets/DEFAULT_RECIPIENT_MNEMONIC
+            DEFAULT_RECIPIENT_PRIVATE_KEY:projects/1027349420744/secrets/DEFAULT_RECIPIENT_PRIVATE_KEY
+            WALLET_MNEMONIC:projects/1027349420744/secrets/WALLET_MNEMONIC
+            WALLET_PRIVATE_KEY:projects/1027349420744/secrets/WALLET_PRIVATE_KEY
+            WALLET_SINGLE_VERIFIED_MNEMONIC:projects/1027349420744/secrets/WALLET_SINGLE_VERIFIED_MNEMONIC
+            WALLET_MULTIPLE_VERIFIED_MNEMONIC:projects/1027349420744/secrets/WALLET_MULTIPLE_VERIFIED_MNEMONIC
+            WALLET_12_WORDS_MNEMONIC:projects/1027349420744/secrets/WALLET_12_WORDS_MNEMONIC
```

Comment:
can the secrets in GCP include a shared prefix (e.g., wallet-e2e-<secret-name>)? I notice they all have a label, but would be nice if there are all grouped together like how we do for repo specific secrets (e.g., cloud-backup-<secret-name>)

Reply:
Added a prefix in 76159a4ac6c50ed4a103705e773b89146c1866f5.